src/text-featurizer/src/test/scala/TextFeaturizerSpec.scala

```diff
@@ -7,12 +7,12 @@ import com.microsoft.ml.spark.schema.DatasetExtensions._
 import org.apache.spark.ml.feature.{NGram, Tokenizer}
 import org.apache.spark.ml.util.MLReadable
 
-class TextFeaturizerSpec extends TestBase with EstimatorFuzzing[TextFeaturizer]{
```

Comment:
Is this just style changes for auto-formatting?

Reply:
yes... I had auto-format on comment turned on in intellij and must have had the file opened.